File: MPI/mpicheck/tests/functions/1.0/MPI_Dims_create.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    int var_0;
    int var_1;
    int var_2[2];
    int ret;
    /* calls */
    ret = MPI_Dims_create(var_0, var_1, var_2);
    ret = PMPI_Dims_create(var_0, var_1, var_2);
    return 0;
}

File: MPI/mpicheck/tests/functions/1.0/MPI_Dims_create.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2(10)
       INTEGER var_3
        call mpi_dims_create(var_0, var_1, var_2, var_3)
       call pmpi_dims_create(var_0, var_1, var_2, var_3)
        end program main
    